content/06.know-your-problem.md

```diff
@@ -1 +1,18 @@
 ## Rule 4: Know your data and your question
+
+In the era of easily accessible datasets, one sometimes starts analyzing data without a good understanding of the study design, namely why the data were collected and how. 
+Having appropriate meta-data, a comprehensive data dictionary, and even the actual data collection protocol is essential for any analysis, including one that involves deep learning. 
+It's even better to also have access to a subject matter expert who has either collected or analyzed this type of data before! 
+For example, if the main reason why the data were collected was to test the impact of an intervention, it may be the case that a randomized controlled trial was performed.
+However, ethical or other study considerations may make this impossible or impractical, in which case the design may have been an observational one, either prospective or retrospective.
+These designs may also incorporate some amount of matching - for example, cases and controls may be selected so that the age range or weight distribution is similar.
+All of these different designs have different assumptions and caveats, which cannot be ignored during a data analysis.
+Many datasets are now passively collected or do not have a specific design, but even in this case it is important to know how individuals or samples were treated (for example, if all samples are from the same study site, if certain ethnic groups or zip codes are oversampled, if there are differences in processing dates or techniques.)
```

Comment:
Not that bio-related, but a potential reference for biased DL/ML systems + addressing the issue 

- Buolamwini, Joy, and Timnit Gebru. "Gender shades: Intersectional accuracy disparities in commercial gender classification." Conference on Fairness, Accountability and Transparency. 2018. (http://proceedings.mlr.press/v81/buolamwini18a.html)